apps/condo/domains/common/utils/tables.utils.ts

```diff
@@ -388,9 +388,7 @@ type TableScrollProps = RcTableProps['scroll'] & { scrollToFirstRowOnChange?: bo
 export const getTableScrollConfig = (isSmall: boolean): TableScrollProps => {
```

Comment:
this function looks strange ... can we remove it?

Reply:
Used all over the place, but I find it a useless function